src/components/CallView/CallView.vue

```diff
@@ -465,6 +479,27 @@ export default {
 			}
 		},
 
+		_handleParticipantRaisedHand(callParticipantModel, raisedHand) {
+			const nickName = callParticipantModel.attributes.name || callParticipantModel.attributes.userId
+			// sometimes the nick name is not available yet...
+			if (nickName) {
+				if (raisedHand) {
+					showMessage(t('spreed', 'Participant {nickName} raised their hand.', { nickName: nickName }))
+				} else {
+					showMessage(t('spreed', 'Participant {nickName} lowered their hand.', { nickName: nickName }))
```

Comment:
I think we shouldn't put out a toast notification for this. You are grabbing attention that someone does not want attention anymore... sounds weird.

Reply:
I also wasn't sure, was thinking of visually impaired people who might want to get notified. But it would likely be overwhelming for them as well.

Removed.